apps/sms/services/README.md

```diff
@@ -0,0 +1,114 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+We'll define different layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Controllers are really tied to a specific view. They'll encapsulate
```

Comment:
But why can't we have this method in ConversationService (aka high-level operation)?